qutebrowser/browser/history.py

```diff
@@ -274,8 +273,6 @@ def add_url(self, url, title="", *, redirect=False, atime=None):
                       (hidden in completion)
             atime: Override the atime used to add the entry
         """
-        if config.get('general', 'private-browsing'):
-            return
```

Comment:
Now we just don't connect the signal anymore, i.e. `add_url` is not called anymore at all when the window is private.